dipy/reconst/shm.py

```diff
@@ -1060,8 +1060,12 @@ def anisotropic_power(sh_coeffs, norm_factor=0.00001, power=2,
         ap += ap_i
         n_start = n_stop
 
-    # Shift the map to be mostly non-negative:
-    log_ap = np.log(ap) - np.log(norm_factor)
+    # Shift the map to be mostly non-negative,
+    # only applying the log operation to non-zero elements
```

Comment:
Yes - somehow the changes I was looking at didn't have your PEP8 commit. Thanks for doing that.

Non-zero would be:

    log_ap[ap != 0] = np.log(ap[ap != 0]) - np.log(norm_factor)

As it is, the code selects for non-negative. But really - this is not crucial - as you pointed up above.